cli/cmd/release_create.go

```diff
@@ -4,13 +4,22 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/manifoldco/promptui"
+	"github.com/replicatedhq/replicated/cli/print"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"gopkg.in/src-d/go-git.v4"
```

Comment:
this should use `https://github.com/go-git/go-git/v5` instead - same API, new home, not an archived github repo